sql/moz-fx-data-shared-prod/monitoring/stable_table_sizes/view.sql

```diff
@@ -4,4 +4,4 @@ AS
 SELECT
   *
 FROM
-  `moz-fx-data-shared-prod.monitoring_derived.stable_table_sizes_v1`
+  `moz-fx-data-shared-prod.monitoring_derived.stable_and_derived_table_sizes_v1`
```

Comment:
It would make sense for this view (`stable_table_sizes`) to only explores stable tables and then add a new view `stable_and_derived_table_sizes` which will return both. Once added the `stable_and_derived_table_sizes` is what we want the Looker dashboard to use.